sdk/tables/Azure.Data.Tables/src/TableClient.cs

```diff
@@ -22,7 +22,7 @@ namespace Azure.Data.Tables
     public class TableClient
     {
         private static readonly char[] ContinuationTokenSplit = { ' ' };
-        private readonly ClientDiagnostics _diagnostics;
+        internal readonly ClientDiagnostics _diagnostics;
```

Comment:
revert?

Reply:
Did it just before this comment showed up :)